googlemock/src/gmock-spec-builders.cc

```diff
@@ -68,6 +68,8 @@ namespace internal {
 // mockers, and all expectations.
 GTEST_API_ GTEST_DEFINE_STATIC_MUTEX_(g_gmock_mutex);
 
+CallReaction intToCallReaction(int mock_behavior);
```

Comment:
Since this function is only called from this file, it does not need a declaration. It only needs to be declared static of be placed in the anonymous namespace to suppress the warning.